### PR TITLE
Polish and proofreading

### PR DESCRIPTION
**Pull request by MadsHeilmann**

### CODE REVIEW

File: src/problem-solution/calculations.tex [hunk 1]
```diff
@@ -1,68 +1,72 @@
...
-Before we can properly print results in the program, we need to calculate the attributes of the different routes.
-This is a relatively difficult task, as we need to calculate the distance, price, \unit{CO_{2}} and comfort.
+Before we can evaluate the routes, we need to calculate the attributes of the different routes.
+This is a relatively difficult task, as we need to calculate the distance, price and \unit{CO_{2}} emission.
 Some attributes are easier to calculate than others, but they all require some form of calculation.
 
 Rejseplanen has systems in place to calculate some of those attributes, but the API does not provide them.
-This means that we have to calculate them ourselves.
-Our calculations will not be 100\% accurate, but they will be close enough for our purposes.
+This means that we have to calculate them ourselves by out own means.
```
Comment (Danar435) [inline src/problem-solution/calculations.tex]: our, not out

File: src/problem-solution/calculations.tex [hunk 2]
```diff
@@ -1,68 +1,72 @@
...
+Our calculations will not be 100\% accurate, as we have simplified some of the calculations, but they will be close
+enough for our purposes.
 
 \subsection{Calculating distance}\label{subsec:calculating-distance}
 
-We need the distance of a route for our calculations.
+We need the distance of a route for our calculations of the route's \unit{CO_{2}} emission.
```
Comment (Danar435) [inline src/problem-solution/calculations.tex]: we use distance for all our calculations

File: src/problem-solution/calculations.tex (hunk 1 above)
Comment (MadsHeilmann) [reply src/problem-solution/calculations.tex]: Now fixed

File: src/problem-solution/calculations.tex (hunk 2 above)
Comment (MadsHeilmann) [reply src/problem-solution/calculations.tex]: Will revert.

**Review by TexWO — APPROVED** [review]

**Review by audio-engineer — APPROVED** [review]

**Pull request merged**